phone/src/apps/settings/components/SettingsApp.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
This isn't referenced anywhere, yeet it.

```suggestion
import React, { useState } from 'react';
```